clusterman/cli/toggle.py

```diff
@@ -48,6 +62,11 @@ def enable(args: argparse.Namespace) -> None:
             'entity': {'S': f'{args.cluster}.{args.pool}.{args.scheduler}'},
         }
     )
+    time.sleep(1)  # Give DynamoDB some time to settle
+    if autoscaling_is_paused(args.cluster, args.pool, args.scheduler, arrow.now()):
+        print('Something went wrong!  The autoscaler is paused')
+    else:
+        print(f'The autoscaler for {args.cluster}.{args.pool}.{args.scheduler} was enabled at {arrow.now()}')
```

Comment:
nit: I think you should use the same `arrow.now()` for both the pause check and the enable message (i.e. set it as a variable).

Reply:
fixed